Nos/Views/BioView.swift

```diff
@@ -32,9 +34,25 @@ struct BioView: View {
 
     private let lineLimit: Int = 5
 
+    private var parsedBio: AttributedString {
+        guard let bio else {
+            return AttributedString()
+        }
+        return NoteParser.parse(
+            content: bio,
+            tags: [[]],
+            context: viewContext
+        )
```

Comment:
@martindsq as I was testing this, I noticed that we don't detect links in the bio for text like `nos.social`. I dug into why and found that the _other_ parse method (the one that returns `(AttributedString, [URL])`) calls extractURLs but _this_ parse method doesn't. Could we add extractURLs to this one (in NoteParser) so we get the updated link detection?

Reply:
Good catch, I didn't realize we have two nearly identical functions

Reply:
I created a new ticket https://github.com/planetary-social/nos/issues/928